### DISCUSSION TITLE
Where does mesa-dist-win keep its log?

### DISCUSSION THREAD

**Discussion by clairem-sl** [pal1000/mesa-dist-win#169] [Q&A] [answered]
I found out that 24.0.1 does not seem to be working properly: Apps start then immediately closes (crashes?)

However, extracting 23.3.6 over 24.0.1 returns my apps in working order.

Where does mesa-dist-win store its logs, if any? I want to see what went wrong.

I used `perappdeploy.cmd` by the way.

**Comment by pal1000** [comment, marked as answer]
There is no specific logging system on Windows but I provide MSVC PDB debug symbols and drop-in replacement MinGW debug binaries to collect call stacks or backtraces respectively in debuggers like MSVC / WinDDbg or GDB respectively. If you can collect such debugging logs please report them directly to [Mesa3D developers](https://gitlab.freedesktop.org/mesa/mesa/-/issues).

**Reply by clairem-sl** [reply]
Okay so I attached WinDbg and this is the result:

https://gist.github.com/clairem-sl/c39dbe3a97bea2b35265c372b0227093

Searching for this ERROR:

```
ERROR:             ICD associated with VkPhysicalDevice does not support GetPhysicalDeviceCalibrateableTimeDomainsKHR
(1438.3768): Security check failure or stack buffer overrun - code c0000409 (!!! second chance !!!)
Subcode: 0x7 FAST_FAIL_FATAL_APP_EXIT 
vulkan_1!vkResetEvent+0xa5c65:
00007ffa`655c4fc5 cd29            int     29h
```

Seems to indicate there had been a regression with 24.0.

It _should_ have been [fixed in 24.0.3](https://gitlab.freedesktop.org/mesa/mesa/-/issues/10513#note_2306358).

So I'm looking forward to the release of 24.0.3 😄

**Reply by clairem-sl** [reply]
Made an issue for upstream btw: https://gitlab.freedesktop.org/mesa/mesa/-/issues/10731
